files/sidebars/gamessidebar.yaml

```
@@ -195,18 +203,19 @@ l10n:
     Audio_for_Web_Games: Audio for Web Games
     2D_collision_detection: 2D collision detection
     Tiles_and_tilemaps_overview: Tiles and tilemaps overview
+    Controls_Gamepad_API: Implementing controls using the Gamepad API
```

Can be made shorter for the sidebar
```suggestion
    Controls_Gamepad_API: Using the Gamepad API
```